src/_pytest/_code/source.py

```diff
@@ -77,17 +77,6 @@ def strip(self):
         source.lines[:] = self.lines[start:end]
         return source
 
-    def putaround(self, before="", after="", indent=" " * 4):
```

Comment:
those methods are potentially breaking accidential  external api, we should document that as removal

Reply:
👍 definitely, I'll make sure to list these out tomorrow in a changelog entry -- already targeting features

Reply:
Maybe give a depreciation grace period so we don't have as many breaks as the recent mark changes?

Reply:
I agree, IMO we should avoid further breakages. 👍 for adding a deprecation warning.